### PR TITLE
hide feedback from UI

### PR DESCRIPTION
**Pull request by JingyuZ**
- we want to get rid of feedback features, but before this, we want to hide it first and see if anyone will notice it or complain about it
- people can acess feedback if they know the URL, but I do not think we need to worry about it right now...

### CODE REVIEW

**Comment by JingyuZ** [comment]
[Get rid of feedback?](https://trello.com/c/VIsaNX3T/21-get-rid-of-feedback)

File: db/schema.rb [hunk 1]
```diff
@@ -49,11 +48,10 @@
     t.string   "nickname"
     t.boolean  "admin",                  default: false
     t.datetime "hidden_at"
+    t.index ["email"], name: "index_users_on_email", unique: true, using: :btree
```
Comment (pronto-saurus) [inline db/schema.rb]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: db/schema.rb [hunk 2]
```diff
@@ -49,11 +48,10 @@
...
+    t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true, using: :btree
```
Comment (pronto-saurus) [inline db/schema.rb]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.

Comment (pronto-saurus) [inline db/schema.rb]: Metrics/LineLength: Line is too long. [110/80]

File: db/schema.rb (hunk 1 above)
Comment (magneland) [inline db/schema.rb]: I'm curious why this file changed in this commit? It looks fine, though.

**Comment by magneland** [comment]
Closing in favor of #40

**Pull request closed without merging**